contributingGuides/STYLE.md

```diff
@@ -492,6 +493,30 @@ const foo = report?.reportID ?? '-1';
 report ? report.reportID : '-1';
 ```
 
+### Extract complex types
+
+All complex types, such as objects and callbacks in function parameters, should be extracted into separate type definitions.
```

Comment:
```suggestion
Complex types, such as objects in function parameters, should be extracted into separate type definitions. Callbacks in function parameters should be extracted if they have possibility to be reused somewhere else.
```

Maybe flex it a little bit? I saw some cases for callbacks that doesn't justify this extraction. [Example here](https://github.com/Expensify/App/blob/main/src/libs/actions/Policy/Category.ts#L1072) and many others that have a simple empty callback function.